compiler/src/dotty/tools/dotc/typer/Namer.scala

```diff
@@ -884,45 +872,18 @@ class Namer { typer: Typer =>
      *  to pick up the context at the point where the completer was created.
      *
      *  If -Yexplicit-nulls, is enabled, we sometimes use the completion context.
-     *  See `ValDefInBlockCompleter`.
      */
-    def completeInContext(denot: SymDenotation, completionContext: Context): Unit = {
+    def completeInContext(denot: SymDenotation, ctx: Context): Unit = {
```

Comment:
nit: could you please update line 1136 or so (the one site that overrides this method) to match the name 'ctx' for the argument (it's currently 'completionContext')?